nipype/interfaces/io.py

```diff
@@ -2863,3 +2864,33 @@ def _list_outputs(self):
 
     def _add_output_traits(self, base):
         return add_traits(base, list(self.inputs.output_query.keys()))
+
+
+class ExportFileInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, desc='Input file name')
+    out_file = File(exists=False, desc='Output file name')
```

Comment:
The default is `False`.

```suggestion
    out_file = File(mandatory=True, desc='Output file name')
```